assets/js/stripe-payment-request.js

```diff
@@ -602,15 +591,25 @@ jQuery( function( $ ) {
 				if ( 0 < paymentRequestError.length ) {
 					evt.preventDefault();
 					window.alert( paymentRequestError );
+					$( document.body ).trigger( 'wc_stripe_unblock_payment_request_button' );
 					return;
 				}
 
-				wc_stripe_payment_request.addToCart();
+				wc_stripe_payment_request.addToCart().then(function (response) {
+					if ( response.error ) {
+						window.alert( response.error );
+						$( document.body ).trigger( 'wc_stripe_unblock_payment_request_button' );
+						return;
+					}
 
-				if ( wc_stripe_payment_request.isCustomPaymentRequestButton( prButton ) || wc_stripe_payment_request.isBrandedPaymentRequestButton( prButton ) ) {
-					evt.preventDefault();
+					paymentRequest.update({
+						total: response.total,
+						displayItems: response.displayItems,
+					});
 					paymentRequest.show();
-				}
+				}).always(function() {
+					$( document.body ).trigger( 'wc_stripe_unblock_payment_request_button' );
+				});
```

Comment:
done 322852442885705abe9a2ea87cbbb62475d66595 

By the way. I see we have a `.prettierrc.js` but no a `prettier` npm task. It would be nice to have one to be able to fix format offenses easily.